packages/@aws-cdk/aws-lambda/lib/lambda-insights.ts

```diff
@@ -40,23 +46,33 @@ export abstract class LambdaInsightsVersion {
   public static fromInsightVersionArn(arn: string): LambdaInsightsVersion {
     class InsightsArn extends LambdaInsightsVersion {
       public readonly layerVersionArn = arn;
+      public _bind(): string {
+        return arn;
+      }
     }
     return new InsightsArn();
   }
 
   // Use the verison to build the object. Not meant to be called by the user -- user should use e.g. VERSION_1_0_54_0
   private static fromInsightsVersion(insightsVersion: string): LambdaInsightsVersion {
 
-    // Check if insights version is valid. This should only happen if one of the public static readonly versions are set incorrectly
-    const versionExists = RegionInfo.regions.some(regionInfo => regionInfo.cloudwatchLambdaInsightsArn(insightsVersion));
-    if (!versionExists) {
-      throw new Error(`Insights version ${insightsVersion} does not exist.`);
-    }
-
     class InsightsVersion extends LambdaInsightsVersion {
       public readonly layerVersionArn = Lazy.uncachedString({
```

Comment:
Not sure if it ok to just remove this? It will only be correct if you are using x86. Also not sure under what circumstances you would want to use this property. From what I can tell it is only used internally.